packages/dx-react-chart-bootstrap4/src/plugins/area-series.jsx

```diff
@@ -8,7 +8,6 @@ export class AreaSeries extends React.PureComponent {
     return (
       <AreaSeriesBase
         seriesComponent={Area}
-        pointComponent={() => null}
         {...this.props}
       />
```

Comment:
The string `AreaSeries.Point = Point` is not expected there. The same in the material-ui theme in all line series (spline, line, area)